### ISSUE TITLE
:use `movies`; throws database not found error

### ISSUE THREAD

**Issue by ikwattro** [neo4j/neo4j-browser#1961]
**Neo4j Version:** 5.15.0
**Operating System:** Any
**API:** Any

The following statement throws `database not found` error although the database name is escaped.

```cypher
:use `movies`;
```

```
A database with the "movies`;" name or alias could not be found.
```

Adding a space before the semicolon works.

### Steps to reproduce
As above

### Expected behavior
Since the database is escaped, Cypher should understand that the semicolon is not part of the database name.

### Actual behavior
database not found error

**Comment by ikwattro** [comment]
@OskarDamkjaer  Yes I got the issue was the semicolon, however it works from cypher-shell so I believe it should work from the browser surface as well. Secondly we generate multi-statements scripts to paste in the browser and it seems the `:use` clause is the only one causing issues with the semicolon.

**Comment by OskarDamkjaer** [comment]
@ikwattro I agree it's a bug we should fix. For your generated scripts I think I adding an extra space can would be a workaround you could use in the meantime:
```
:use `movies` ;
create ()
``` 

The above worked when I tried it just now.
